src/components/Header.tsx

```diff
@@ -62,8 +62,17 @@ const Header = () => {
             </Dropdown.Header>
             <Dropdown.Item>Settings</Dropdown.Item>
             <Dropdown.Divider />
-            <Dropdown.Item>Sign out</Dropdown.Item>
+            <Link href="/add-listing">
+              <Dropdown.Item>Add Listing</Dropdown.Item>
+            </Link>
             <Dropdown.Divider />
+            <Dropdown.Item>Sign out</Dropdown.Item>
+            <Link href="/log-in">
+              <Dropdown.Item>Log In</Dropdown.Item>
+            </Link>
```

Comment:
do we need the login/log out here? wouldn't you just do one or the other?

Reply:
i changed this in the other pr